shallow_backup.py

```diff
@@ -689,6 +697,13 @@ def cli(complete, dotfiles, configs, packages, fonts, old_path, new_path, remote
 		print(Fore.RED + Style.BRIGHT +
 		      "Removed config file..." + Style.RESET_ALL)
 		sys.exit()
+	elif clean:
+		backup_home_path = get_config()["backup_path"]
+		if prompt_yes_no("Erase backup directory {}?".format(backup_home_path), Fore.RED):
```

Comment:
I've gone back and forth about actually prompting here. If we get to this point in the program, the user either used the `-destroy_backup` flag, or hit "Destroy backup" in the menu. That's a pretty deliberate action.

I'm not entirely sure we need to prompt here. What are your thoughts?

**Side thoughts**

1. Let's change the flag to `-destroy_backup` since that's a lot clearer.
2. Then we should refactor the other method names as needed.

Reply:
@alichtman Do you mean removing the prompt and just go straightly on the delete operation? My thought was that for any dangerous action, we have to double confirm it to user that is why I added the prompt. Final thoughts?

For your side thoughts no. 3, do you mean adding it after this?

```python
elif selection == "reinstall configs":
	reinstall_config_files(configs_path)
# Add here
elif selection == "Destroy backups":
      destroy_backup()
      # Can you advise if we need to exit at this point or we can go on the git stuffs below?
      sys.exit()

git_add_all_commit(repo, backup_home_path)
git_push_if_possible(repo)
sys.exit()
```

Reply:
> My thought was that for any dangerous action, we have to double confirm it to user that is why I added the prompt.

Alright, I have a good solution. We only prompt if they select to remove the backup from the CLI. If they remove with the flag, do that without a double confirmation. So, the process will be slightly different based on how the user selects to remove the backup.

> For your side thoughts no. 3, do you mean adding it after this?

I meant that we should add it to the menu of options people can choose from in the CLI.